tests/examples/test_plan.py

```diff
@@ -18,19 +18,34 @@
 from pathlib import Path
 
 BASE_PATH = Path(__file__).parent
-COUNT_TEST_RE = re.compile(r'# tftest +modules=(?P<modules>\d+) +resources=(?P<resources>\d+)' +
-                           r'(?: +files=(?P<files>[\w@,_-]+))?' +
-                           r'(?: +inventory=(?P<inventory>[\w\-.]+))?')
+COUNT_TEST_RE = re.compile(
+    r'# tftest +modules=(?P<modules>\d+) +resources=(?P<resources>\d+)' +
+    r'(?: +files=(?P<files>[\w@,_-]+))?' +
+    r'(?: +fixtures=(?P<fixtures>[\w@,_/.-]+))?' +
+    r'(?: +inventory=(?P<inventory>[\w\-.]+))?')
 
 
-def prepare_files(example, test_path, line):
-  if line is not None:
-    requested_files = line.split(',')
+def prepare_files(example, test_path, files, fixtures):
+  if files is not None:
+    requested_files = files.split(',')
     for f in requested_files:
       destination = test_path / example.files[f].path
       destination.parent.mkdir(parents=True, exist_ok=True)
       destination.write_text(example.files[f].content)
 
+  if fixtures is not None:
+    requested_fixtures = fixtures.split(',')
+    for f in requested_fixtures:
+      if f.startswith('fixtures/'):
+        # fixture is specified referencing a global fixture in (tests/fixtures/)
```

Comment:
@ludoo Here is a way to define centralized fixtures.

Reply:
Can we default to that, and use tfdoc like Julio and I were discussing to inject the link to the fixtures used (or an expandable box) in the README?

Reply:
done. Updated tfdoc to list the used fixtures after the outputs section